### ISSUE TITLE
Making two lines

### ISSUE THREAD

**Issue by MichalNemec** [SirusCodes/morphing_text#6]
Hello,
Is it possible to make text two lines or more? 
"test\n test" doesnt work properly with ScaleMorphingText

**Comment by SirusCodes** [comment]
Currently, it is not possible as it can only handle 1D transitions.
We can add 2D transitions by changing [_calculateMove](https://github.com/SirusCodes/morphing_text/blob/3a4de0bdb1aa78ba1597b626ae071487aa8aa269/lib/src/core/custom_morphing_text.dart#L157) move and adding z-axis.

Things in my mind are how should we approach it.
1. Only allow chars to move in the same line _or_ allow them to move across lines
2. How should be handled if a line is increased or decreased?

Also if you can add a PR that would be great.

**Comment by MichalNemec** [comment]
I see that its more complicated.
Could you introduce another ScaleMorphingText which would be just fadein fadeout, where the two or more lines could be present. I am completely fine with fadein/fadeout animation. I was trying bunch of packages and only this fit my bill regarding functionality. It would also be nice to have some linebreak on its own, because line when its longer and fontsize i.e. 50, its cut off.

**Comment by SirusCodes** [comment]
Have you seen [animated_text_kit](https://pub.dev/packages/animated_text_kit)? It might work for you.

**Comment by MichalNemec** [comment]
yes, tried it, looks nice, but last item isnt staying put, no callback upon completing, oneliner/cutting text.

**Comment by SirusCodes** [comment]
[FadeTransition](https://api.flutter.dev/flutter/widgets/FadeTransition-class.html) should help you with fadein and fadeout animations.

Also for further conversation, I feel it's better to continue on my [twitter](https://twitter.com/SirusTweets) or [other platform](https://links.darshanrander.com) as it is going a bit off topic

**Comment by MichalNemec** [comment]
thanks, ill try to make it by myself somehow, when i have time
